<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sucesso</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(9, 38, 66);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 118, 209);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>SAIR DO WINDOWS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 149, 0);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Sucesso!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="verificado">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 90pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="informacao_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Seu pc vai ser ? em:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="informacao_h">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00 : 00 : 00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Até mais!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_finalizar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 149, 0);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Até mais!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(6, 26, 44);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>389</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Bloquear o usuário ao sair</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
